### PR TITLE
Change color

### PR DESCRIPTION
**Pull request by Antsthebul**
Created a base class for dungeon items/monsters and characters to inherit from. Added reset method to dungeon item

### CODE REVIEW

File: src/resources/base.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from rich.text import Text
+
+
+class DungeonBase:
```
Comment (n0remac) [inline src/resources/base.py:4]: This might be better named as `AbstractDungeonEntity`

File: src/resources/items.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+from .base import DungeonBase
+from .character import Character
+
+
+class DungeonItem(DungeonBase):
```
Comment (n0remac) [inline src/resources/items.py:5]: This should be named `DungeonItems`

Comment (n0remac) [reply src/resources/items.py:5]: or maybe `ItemManager`

You can put a draw function in here as well that draws all the items in the list.

Comment (n0remac) [reply src/resources/items.py:5]: If this is a manager it doesn't need to extend DungeonBase

File: src/resources/items.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    """Dungeon Items that change players color if captured"""
+
+    used_items = []
+    unused_items = []
```
Comment (n0remac) [inline src/resources/items.py:9]: If this is meant to be an abstract class for all items instead of a manager then remove these lists.

Comment (Antsthebul) [reply src/resources/items.py:9]: Ill remove it for now but my idea was this was just for the color changing items. I was thinking of making a factory where we can simply pass in location, symbol and color. That way things dont get repeated

**Pull request closed without merging**